templates/ingress.yaml

```diff
@@ -16,26 +16,30 @@ spec:
   tls:
     - secretName: {{ default .Values.ingress.secretName .Values.ingress.secret.name}}
       hosts:
-        - {{ printf "*.%s" .Values.config.rootDomain | quote }}
```

Comment:
I have mixed feelings about ditching the wildcard domain here. This would result in each managed host needing a change not only to policy, but also to the ingress. Thoughts @travisgroth ?

Reply:
I understand some users might choose to manage the DNS records in another way. But proxy already mandates to have the list of `config.policy.[].from`.